modules/internal/ChapelBase.chpl

```diff
@@ -1539,7 +1529,7 @@ module ChapelBase {
   pragma "no doc"
   inline proc _defaultOf(type t) where t: _sync_aux_t return _nullSyncVarAuxFields;
   pragma "no doc"
-  inline proc _defaultOf(type t) where t == _task_list return _nullTaskList;
+  inline proc _defaultOf(type t) where t == c_void_ptr return _nullVoidPtr;
```

Comment:
Ah, didn't spot that. Things seem to work fine with this line removed, so I'll do another commit to take it out.